sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/NetworkStateHelper.java

```diff
@@ -205,8 +205,7 @@ private synchronized void onNetworkLost(Network network) {
         Network[] networks = mConnectivityManager.getAllNetworks();
```

Comment:
No need to use synchronized anymore in that method as well.

Reply:
Didn't hear any answers for my previous comment, @guperrot can you explain why we don't need this `synchronized`?